proto/ethermint/evm/v1/evm.proto

```diff
@@ -28,6 +28,24 @@ message Params {
   repeated string evm_channels = 8 [(gogoproto.customname) = "EVMChannels"];
   // access_control defines the permission policy of the EVM
   AccessControl access_control = 9 [(gogoproto.customname) = "AccessControl", (gogoproto.nullable) = false];
+  // CustomOpcodes is the list of opcodes to update
+  repeated CustomOpCode custom_opcodes = 10 [(gogoproto.customname) = "CustomOpCodes", (gogoproto.nullable) = false];
+}
+
+message CustomOpCode {
+  // opcode is the opcode to update
+  OpCode op_code = 1 ;
```

Comment:
Since the type `OpCode` is used also in go-ethereum to refer to `byte`, I would use another name to avoid confusion